sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -85,13 +85,12 @@ repos:
     rev: v1.26.2
     hooks:
       - id: pyupgrade
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v1.9.4
-    hooks:
-      - id: seed-isort-config
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
+  - repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21-2
     hooks:
       - id: isort
         name: isort except __init__.py
         exclude: /__init__\.py$
+        # TODO, the next time we upgrade isort's rev, we should remove 'types'
+        types: [python]
+        additional_dependencies: ["pipreqs==0.4.10", "pip-api==0.0.13"]
```

Comment:
Don't you need to add requirements?

See https://github.com/timothycrosley/isort#installing-isort

Reply:
`requirements` is a sort of alias on `pipreqs` + `pip-api`: https://github.com/timothycrosley/isort/blob/0882c8885b88618ea55b97ace256cdf833a1547d/pyproject.toml#L42